test/integration/idle_timeout_integration_test.cc

```diff
@@ -25,7 +25,14 @@ class IdleTimeoutIntegrationTest : public HttpProtocolIntegrationTest {
             hcm.mutable_request_timeout()->set_seconds(0);
             hcm.mutable_request_timeout()->set_nanos(RequestTimeoutMs * 1000 * 1000);
           }
-
+          if (enable_sleeping_filter_) {
+            config_helper_.addFilter(fmt::format(R"EOF(
+name: sleeping-filter
+config:
+  sleep_duration: {}ms
+)EOF",
```

Comment:
Did this pass the fix_format script? Indentation seems weird.

Reply:
Yes